### PR TITLE
DeQueue using Array in C++

### PR DESCRIPTION
**Pull request by imsushant12**
Related Issue: #237
Related PR: #1535

### CODE REVIEW

**Comment by imsushant12** [comment]
@krishna-NIT

File: Code/C++/dequeue_using_array.cpp [hunk 1]
```diff
@@ -0,0 +1,328 @@
+/*
+DeQueue or Double Ended Queue is a type of queue in which insertion
+and removal of elements can be performed from either front or rear.
+*/
```
Comment (krishna-NIT) [inline Code/C++/dequeue_using_array.cpp:4]: give brief

Comment (imsushant12) [reply Code/C++/dequeue_using_array.cpp:7]: done, please review

**Comment by krishna-NIT** [comment]
@rish-singhal LGTM

**Comment by imsushant12** [comment]
@rish-singhal  please review

**Comment by imsushant12** [comment]
@rish-singhal , sir what about the level - labels?

**Comment by rish-singhal** [comment]
thanks for reminding me! : D

**Pull request merged**